README.md

```diff
@@ -292,11 +292,18 @@ To enable set `elasticsearch.enabled=true` in your `values.yaml` and deploy the
 For each instance you would like to enable this on, set the configuration values in the respective `config.yml`:
 
 ```yaml
-K8S_HARMONY_ENABLE_SHARED_HARMONY_SEARCH: true
 RUN_ELASTICSEARCH: false
+K8S_HARMONY_ENABLE_SHARED_SEARCH_CLUSTER: true
 ```
 
-* And create the user on the cluster with `tutor k8s harmony create-elasticsearch-user`.
+* Create the user on the cluster with `tutor k8s harmony create-elasticsearch-user`.
+* Copy the Elasticsearch CA certificate to the instance's namespace where `$INSTANCE_NAMESPACE` is where the instance is installed in.
+  ```shell
+  kubectl get secret "search-cluster-certificates-elasticsearch" -n "harmony" -o "yaml" | \
+      grep -v '^\s*namespace:\s' | \
+      sed s/-elasticsearch//g |\
+      kubectl apply -n "$INSTANCE_NAMESPACE" --force -f -
```

Comment:
We do not have a created namespace at this point yet. 
Should we mention this in the documentation?